sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/JsonSerializable.java

```diff
@@ -32,14 +33,14 @@
     private final static ObjectMapper OBJECT_MAPPER = Utils.getSimpleObjectMapper();
     private ObjectMapper om;
     transient ObjectNode propertyBag = null;
-    
```

Comment:
Please refrain from making unrelated formatting changes, you can configure intellij to stop doing this automatically

Reply:
I really wanted to, but couldn't stop it. :( 
You can show me a demo sometime :)